pkg/client/unversioned/clientcmd/config.go

```diff
@@ -408,7 +427,9 @@ func writePreferences(configAccess ConfigAccess, newPrefs clientcmdapi.Preferenc
 func GetConfigFromFileOrDie(filename string) *clientcmdapi.Config {
 	config, err := LoadFromFile(filename)
 	if err != nil && !os.IsNotExist(err) {
-		glog.FatalDepth(1, err)
+		glog.ErrorDepth(1, err)
```

Comment:
Done. `GetConfigFromFileOrDie` should be unchanged. `ModifyConfig` now uses the new `getConfigFromFile` and handles its errors.